Morcoso_Daisy/script.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded", function () {
+  let nameInput = document.getElementById("name");
+  let commentTextarea = document.getElementById("textarea_for_comment");
+  let commentButton = document.getElementById("comment");
+
+  nameInput.addEventListener("input", checkFormValidity);
+  commentTextarea.addEventListener("input", checkFormValidity);
+  commentButton.addEventListener("click", addComment);
+
+  document.getElementById("sort_asc").addEventListener("click", function () {
+    sortComments("asc");
+  });
+  document.getElementById("sort_desc").addEventListener("click", function () {
+    sortComments("desc");
+  });
+
+  loadComments();
+});
+
+let comments = [];
+
+function checkFormValidity() {
+  let nameValue = document.getElementById("name").value.trim();
+  let commentValue = document
+    .getElementById("textarea_for_comment")
+    .value.trim();
+  document.getElementById("comment").disabled = !(nameValue && commentValue);
+}
+
+function addComment() {
+  const nameInput = document.getElementById("name").value;
+  const commentInput = document.getElementById("textarea_for_comment").value;
+  if (nameInput.trim() && commentInput.trim()) {
+    const comment = {
+      name: nameInput,
+      text: commentInput,
+      date: new Date().toISOString(),
+    };
+    comments.push(comment);
+    saveComments();
+    displayComments();
+    document.getElementById("name").value = "";
+    document.getElementById("textarea_for_comment").value = "";
+  }
+}
+
+function displayComments() {
+  const commentsSection = document.querySelector(".comments-of-team-comment");
+  commentsSection.innerHTML = "<h3>Comments</h3>";
+  comments.forEach((comment) => {
+    const commentDiv = document.createElement("div");
+    commentDiv.classList.add("comment");
+    const commentDate = new Date(comment.date).toLocaleString();
+    commentDiv.innerHTML = `
+        <p>Name: ${comment.name}</p>
+        <p>Comment: ${comment.text}</p>
+        <p class="comment-date">Date: ${commentDate}</p>
+    `;
+    commentsSection.appendChild(commentDiv);
+  });
+}
+
+function sortComments(order) {
+  if (order === "asc") {
```

Comment:
Please follow this guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g294850e169b_0_1